html/schedule.html

```diff
@@ -126,44 +126,46 @@
           <p>{{imageProperties.text}}</p>
         </div>
       </template>
-      <template is="dom-repeat" id="entryList" items="{{entries}}">
-        <div class="card" layout="" vertical="" xcenter="" on-tap="toggleExpand">
-          <div class="card-header" layout="" horizontal="" center="">
-            <img src="{{item.avatar}}">
-            <div class="card-mid" layout="" vertical="">
-              <p item="time"><span>{{item.startTime}}</span> - <span>{{item.endTime}}</span>
-                <template is="dom-if" if="{{hasTeacherAndRoom(item)}}">
-                  (<a on-tap="teacherThunk" item="link" href="#">{{item.teacherLastName}}</a>,
-                  <a on-tap="roomThunk" item="link" href="#">{{item.room}}</a>)
-                </template>
-                <template is="dom-if" if="{{hasTeacherOnly(item)}}">
-                  (<a on-tap="teacherThunk" item="link" href="#">{{item.teacherLastName}}</a>)
-                </template>
-                <template is="dom-if" if="{{hasRoomOnly(item)}}">
-                  (<a on-tap="roomThunk" item="link" href="#">{{item.room}}</a>)
-                </template>
-              </p>
-              <h2>{{item.name}}</h2>
+      <template is="dom-if" if="{{not(imageProperties)}}">
```

Comment:
Instead of using imageProperties, you might want to instead use |entries|, and have a single class with time 8:00-3:15, no teacher/room, and avatar pointing to the image URL.

That might be more consistent.